nodejs/dist/uws.js

```diff
@@ -289,6 +289,20 @@ class Server extends EventEmitter {
         this._upgradeListener = null;
         this._noDelay = options.noDelay === undefined ? true : options.noDelay;
         this._lastUpgradeListener = true;
+        this._abortConnection = (socket, code, name) => {
```

Comment:
https://nodejs.org/api/http.html#http_response_writehead_statuscode_statusmessage_headers

writeHead formats the status stuff, then you just call end() to shutdown.